content/app/development/configuration/partytype/_index.en.md

```diff
@@ -3,22 +3,21 @@ title: Party types
 description: How to ensure that only specific types of party types can use an app.
 toc: true
 weight: 150
-tags: [translate-to-english]
 ---
 
-## Avgrense tilgang i Altinn Studio designer
+## Limit access in Altinn Studio designer
```

Comment:
```suggestion
## Limit access using Altinn Studio designer
```
I suggest changing the headings a bit to make it clear that this approach and the one below are two different alternatives to reach the same goal.